Assets/MixedRealityToolkit/Utilities/Editor/Preferences/EditorPreferences.cs

```diff
@@ -98,7 +98,7 @@ public static float Get(string key, float defaultValue)
         /// <summary>
         /// Get the saved <see cref="int"/> from the <see href="https://docs.unity3d.com/ScriptReference/EditorPrefs.html">EditorPrefs</see>.
         /// </summary>
-        public static int Get(string key, int defaultValue)
+        private static int Get(string key, int defaultValue)
```

Comment:
Everything else in this class stayed `public`. Why'd this one change?

Reply:
```suggestion
        public static int Get(string key, int defaultValue)
```